CHANGELOG

```diff
@@ -1,3 +1,72 @@
+== OpenDataPlane (1.19.0.0)
```

Comment:
if we do not change API here it has to be 1.18.0.2

Reply:
OK.